ansible/playbooks/roles/repository/files/download-requirements/requirements/x86_64/ubuntu.yml

```diff
@@ -0,0 +1,420 @@
+---
+packages:
+  - name: 'adduser'
+    sha256: 5f7ea9d1d52a2a9c349468f89d160230e21c8542faed1b1a97c23bce873e17b4
```

Comment:
I've checked that:
- apt download
- apt-get download
- aptitude download

are not able to detect (re-download) corrupted package unless it has different size.

Example 1:

```
root@vm-01:~# sha256sum wget_1.20.3-1ubuntu2_amd64.deb
9a656b8746a59983c1ddfc802f664af302161980b6b03353c5fea3e32cb65959

# Change the fourth byte to 0xFE
root@vm-01:~# echo -ne \\xFE | dd conv=notrunc bs=1 count=1 seek=3 of=wget_1.20.3-1ubuntu2_amd64.deb
1 byte copied, 3.56e-05 s, 28.1 kB/s

root@vm-01:~# sha256sum wget_1.20.3-1ubuntu2_amd64.deb
0d54fdab59ab251952a1f76d739f0958195240b10c29ae5128e93bf7a17a2c0a

root@vm-01:~# apt-get download wget

root@vm-01:~# sha256sum wget_1.20.3-1ubuntu2_amd64.deb
0d54fdab59ab251952a1f76d739f0958195240b10c29ae5128e93bf7a17a2c0a
```

However it's possible to get expected sha256:

Example 2:

```
root@vm-01:~# apt-cache show wget | grep SHA256 | head -1 | awk '{print $2}'
9a656b8746a59983c1ddfc802f664af302161980b6b03353c5fea3e32cb65959
```